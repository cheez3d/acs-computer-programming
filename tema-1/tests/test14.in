o suta douazeci si noua
2224446622244406444#44408777334440777788#833
trei patru noua patru noua
2
patru opt
noua sase
unu doi trei
