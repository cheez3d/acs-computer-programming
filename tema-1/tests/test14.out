6660777788#820366688299993322244407777444066#666882
cinci mii trei sute
877733444066678077772777733066678077772777733
3
886688 3666444 877733444 
877733444 3
0
0
